### ISSUE TITLE
how to set mean value and std value?

### ISSUE THREAD

**Issue by LiYijin** [hujie-frank/SENet#53]
Hello, What is senet mean value and std value? Thanks a lot !

**Comment by LiYijin** [comment]
I have known that the synset_words link is as follows according to another issue:
https://github.com/Cadene/pretrained-models.pytorch/blob/master/data/imagenet_synsets.txt

**Comment by elesun2018** [comment]
Check failed: labels_.size() == output_layer->channels() (1862 vs. 1000) Number of labels is different from the output layer dimension.

**Comment by hujie-frank** [comment]
The mean values are written at the beginning of each [model](https://github.com/hujie-frank/SENet/tree/master/models) (i.e. 104, 117, 123 for BGR channels). And we don't set std value. @LiYijin
